ffi/Cargo.toml

```diff
@@ -6,7 +6,7 @@ license = "Apache-2.0"
 authors = ["Valkey GLIDE Maintainers"]
 
 [lib]
-crate-type = ["staticlib", "rlib"]
+crate-type = ["staticlib", "rlib", "cdylib"]
```

Comment:
we're going to link it probably as a static library, so be aware that it will be changed later on - but that's ok to leave this change for now